lls/cache.c

```diff
@@ -311,6 +312,22 @@ lls_process_reqs(struct lls_config *lls_conf)
 		case LLS_REQ_PUT:
 			lls_process_put(lls_conf, &reqs[i]->u.put);
 			break;
+		case LLS_REQ_ARP: {
+			struct lls_arp_req *arp = &reqs[i]->u.arp;
+			uint16_t tx_queue =
+				(arp->iface == &lls_conf->net->front)
+				? lls_conf->tx_queue_front
+				: lls_conf->tx_queue_back;
+			int i;
+			for (i = 0; i < arp->num_pkts; i++) {
+				if (process_arp(lls_conf, arp->iface,
+						tx_queue, arp->pkts[i],
+						rte_pktmbuf_mtod(arp->pkts[i],
+						    struct ether_hdr *)) == -1)
```

Comment:
Shouldn't we keep this line aligned with the above one?

Reply:
Since there is a nested function call (to `rte_pktmbuf_mtod()`), I chose to indent its parameters to make it clearer that they are parameters for that function call, and not parameters for `process_arp()`.